docs/docs/resources/pipeline_config/config.yaml

```diff
@@ -0,0 +1,55 @@
+# CONFIGURATION
+#
+# The path to the folder containing the defaults file and the environment
+# defaults files.
+# REQUIRED
+defaults_path: /path/to/defaults
+# The environment you want to generate and deploy the pipeline to. Suffix your
+# environment files with this value (e.g. defaults_development.yaml for
+# environment=development).
+# REQUIRED
+environment: development
+# The kafka broker address.
+# REQUIRED
+broker: "http://localhost:9092"
+# The name of the defaults file and the prefix of the defaults environment file.
+defaults_filename_prefix: defaults
+# Base class for settings, allowing values to be overridden by environment
+# variables. This is useful in production for secrets you do not wish to save in
+# code, it plays nicely with docker(-compose), Heroku and any 12 factor app
+# design.
```

Comment:
where did you get this info from?

Reply:
Description of the fields in the schema :)

Reply:
I can't make sense of this. what does it even belong to? do you know? it definitely doesn't belong to `topic_name_config`

Reply:
Belongs to `BaseSettings`, I added docstrings to the classes to override it (and to describe them a bit).

Reply:
great, thank you!